css/src/components/banner.scss

```diff
@@ -0,0 +1,100 @@
+$banner-background: $secondary-background !default;
+$banner-padding: $spacer-5 !default;
+
+$banner-font-size: $font-size-7 !default;
+$banner-color: $text !default;
+
+$banner-border-color: $border-white-high-contrast !default;
+$banner-border-width: $border-width !default;
+
+.banner {
+	@extend %layout-gap;
+
+	display: block;
+	position: relative;
+	outline-color: $text;
+	background-color: $banner-background;
+	color: $banner-color;
+	font-size: $banner-font-size;
+	padding-block: $banner-padding;
+	word-wrap: break-word;
+	word-break: break-word;
+	border-block: $banner-border-width solid $banner-border-color;
```

Comment:
In the high contrast theme as well as in an operating system forced-colors contrast theme, I think the lack of left/right borders kind of stands out.

**High contrast theme:**
<img width="1041" alt="Banner in Usage section viewed in the high contrast theme. The banner has a black background, like the rest of the page. It has white borders on the top and bottom." src="https://user-images.githubusercontent.com/18060369/230116748-b5c95f98-e8ad-4652-b932-e5020d8a36cd.png">

**Forced colors:**
![Banner in Usage section viewed in the Dusk forced colors theme. The banner has a gray background, like the rest of the page. It has white borders on the top and bottom.](https://user-images.githubusercontent.com/18060369/230117165-58eb4a08-131c-45f3-8910-d03065beadfe.png)

I think it might be nice to add left/right borders for the high contrast theme, as well as in forced colors mode. Thoughts?

Reply:
On further thinking, this could just be an artifact of the way I'm viewing this not at fullscreen. A full-width banner wouldn't really need to show left/right borders.

That in mind, feel free to resolve this if you don't think we need this.

Reply:
That was my thinking - since it's a full-width component that takes the whole screen left/right borders felt redundant to me.